### DISCUSSION TITLE
Multi channel edits

### DISCUSSION THREAD

**Discussion by DepreTux** [bangcorrupt/hacktribe#52] [Ideas]
It would be nice to be able to edit the same parameter on multiple pads with a single knob twist.

For example, changing the filter cutoff on pads 1-8.

That would allow for truly polyphonic patches.

Thanks for considering it!

**Comment by bangcorrupt** [comment]
I'd like to add parameter linking at some point, but anything that can be achieved via MIDI is low priority.

**Reply by ptowns** [reply]
Yes, very easy with simple midi work. I use pure data, but via USB midi cc to channels 2,3,4..... done, but splitting notes is a bit harder.
Would be a bit of a finger strain, but would be cool if you could hold trigger and select multiple pads to control at once! Like the solo mode on the part mute button.

**Comment by ecco00** [comment]
a great and much needed change: trigger+padnum selecting different parts to change on the fly parameters like volume, filter cutoffs etc would be of huge interest, i definately agree!!

**Comment by bukru** [comment]
a keybord mode triggering multiple parts would be awesome to make multioscillator instrument.
I do that with external gear to copie note on multiple channel, but it require external gear and configuration.
BUT actually, i don't understand why my midi keyboard (oxygen mini pro) can't send note on multiple channel, i don't understand fabricants...
